### PR TITLE
split automation tests to run in parallel in separated packages

### PR DESCRIPTION
**Pull request by AvivCX**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](https://github.com/checkmarx-ts/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/checkmarx-ltd/open-source-template/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

split automation tests to run in parallel in separated packages
this will give better visibility on the tests status, and reduce the overall duration of the integration tests

### Testing

separated the integration tests to 4 different packages
all integration tests and end to end tests passed successfully multiple times

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).  *If documentaiton is a Wiki Update, please indicate desired changes within PR MD Comment*
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by AvivCX** [comment]
@alex-ko-dev @ofersk @NimrodGolan @udiraz2 @olgakil 
see the new structure of integration tests here: https://app.circleci.com/pipelines/github/checkmarx-ltd/cx-flow?branch=pr-split-sca-integration-tests
All ci tests passed succesfully few times with failures

**please confirm the right tests run under each package**

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010429&projectid=150)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|362
Low|3907
Informational|54

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

File: build-11.gradle [hunk 1]
```diff
@@ -123,6 +123,11 @@ task integrationTest(type: Test) {
         systemProperty "testprofile", "integration"
     }
 }
+task ScaIntegrationTest(type: Test) {
+    useJUnitPlatform() {
+        systemProperty "testprofile", "scaIntegration"
+    }
+}
```
Comment (ofersk) [inline build-11.gradle:131]: don't we need all the sections we added to the java8 build?

Comment (AvivCX) [reply build-11.gradle:131]: no, because no one runs the tasks on the other build.gradle except the main one
I will remove this one also from here

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010431&projectid=150)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|362
Low|3907
Informational|50

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010432&projectid=150)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|362
Low|3907
Informational|50

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010434&projectid=150)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|332
Low|3907
Informational|50

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

**Pull request merged**